cmd/agent/dist/conf.d/cisco_sdwan.d/conf.yaml.example

```
@@ -0,0 +1,76 @@
+## This file is overwritten upon Agent upgrade.
+## To make modifications to the check configuration, please copy this file
+## to `conf.yaml` and make your changes on that file.
+
+instances:
+
+  -
+    ## @param vmanage_endpoint - string
+    ## The IP address/endpoint of the Cisco Manager instance.
+    #
+    # vmanage_endpoint: vmanage.example.com
```

For required parameter, we tend to use the following syntax:

```suggestion
    # vmanage_endpoint: <VMANAGE_ENDPOINT>
```

---

Good call, updated in d243d21